documentation/book/ref-storage-ephemeral.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// assembly-storage.adoc
+
+[id='ref-ephemeral-storage-{context}']
+= Ephemeral storage
+
+Ephemeral storage is using the {K8sEmptyDir} to store tha data.
```

Comment:
is using the {K8sEmptyDir} to store > uses the {K8sEmptyDir} to store
tha data. >  data.